src/pages/auth/utils/authHandlers.js

```diff
@@ -42,6 +42,7 @@ export const signUpHandler = async (
       setUser(response.data.createdUser);
       setEncodedToken(response.data.encodedTokern);
       localStorage.setItem("encodedToken", response.data.encodedToken);
```

Comment:
usually instead of local staorage we will use cookies becuase it have expiry durations that we commonly see in bank apps and in other trasactional web apps

Reply:
Yeah, for now could switch to session storage, as the session data would destroy once the user closes the window.